ebos/eclinterregflows.cc

```diff
@@ -55,7 +55,7 @@ addConnection(const Cell& source,
     }
 
     if (! source.isInterior ||
-        (source.cartesianIndex > destination.cartesianIndex))
+        (source.activeIndex > destination.activeIndex))
```

Comment:
This one got lost in the review:
This change looks strange. Might this be the cause for the failing test?

Reply:
I don't remember why it got changed. I'll turn it into the old `cartesianIndex` then. Thanks!